parts/kubernetesmasteraddons-kubernetes-dashboard-deployment.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     kubernetes.io/cluster-service: "true"
     k8s-app: kubernetes-dashboard
-    version: v1.5.1
```

Comment:
Huh. Looks like it's no longer in the upstream yaml. https://github.com/kubernetes/dashboard/blob/master/src/deploy/kubernetes-dashboard.yaml Sounds good to me.